adapters/pubmatic.go

```diff
@@ -150,8 +218,24 @@ func (a *PubmaticAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder
 				Height:      bid.H,
 				DealId:      bid.DealID,
 			}
-			bids = append(bids, &pbid)
+			pbid.CreativeMediaType = "banner"
+
+			bidInfo[pbid.AdUnitCode] = append(bidInfo[pbid.AdUnitCode], &pbid)
+
+			glog.Infof("[PUBMATIC] Received Bid for PubID [%s] AdUnit [%s] BidID [%s] Size [%dx%d] Price [%f] \n",
+				pubId, pbid.AdUnitCode, pbid.BidID, pbid.Width, pbid.Height, pbid.Price)
+		}
+	}
+
+	bids := make(pbs.PBSBidSlice, 0)
+	for _, bids_list := range bidInfo {
+
+		if len(bids_list) > 1 {
+			sort.Sort(bids_list)
 		}
+		bids = append(bids, bids_list[0])
```

Comment:
The adapter shouldn't need to do this. The publisher's request actually has a flag which determines whether or not the bids are sorted, and [that happens centrally](https://github.com/prebid/prebid-server/blob/master/pbs_light.go#L433) once all bidders have responded with all their bids.

Reply:
Code updated to return all bids of same adunit code from PubMatic adapter